src/SignalRServiceExtension/Config/SignalRConfigProvider.cs

```diff
@@ -4,19 +4,23 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
 using Microsoft.Azure.SignalR.Management;
 using Microsoft.Azure.WebJobs.Description;
 using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.Azure.WebJobs.Host.Config;
+using Microsoft.Azure.WebJobs.Logging;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
 {
-    [Extension("SignalR")]
-    internal class SignalRConfigProvider : IExtensionConfigProvider
+    [Extension("SignalR", "signalr")]
```

Comment:
>signalr [](start = 27, length = 7)

Is this case sensitive? Is there better compatible ways instead of list all? how about `signalR` which also might be a usual naming convention way?

Reply:
The second parameter is used in Url path(runtime/webhook/signalr), so I think lower case makes sense.

Reply:
Get it. So in RT does the url accepts lowercase only? Could it somehow be modified to not normalized ones and be blocked here?

---
In reply to: [385643266](https://github.com/Azure/azure-functions-signalrservice-extension/pull/94#discussion_r385643266) [](ancestors = 385643266)

Reply:
No, RT can accept any case because customer may not use Azure Function. But for the trigger, from customer's view, the url is generate by trigger and ask them to copy/paste. I think it's OK.

Reply:
Good.

---
In reply to: [385649396](https://github.com/Azure/azure-functions-signalrservice-extension/pull/94#discussion_r385649396) [](ancestors = 385649396)